lib/OpenQA/Worker.pm

```diff
@@ -37,15 +37,14 @@ sub init {
     $instance  = $options->{instance} if defined $options->{instance};
     $pooldir   = $OpenQA::Utils::prjdir . '/pool/' . $instance;
     $nocleanup = $options->{"no-cleanup"};
-    $verbose   = $options->{verbose} if defined $options->{verbose};
 
     my $logdir = $ENV{OPENQA_WORKER_LOGDIR} // $worker_settings->{LOG_DIR};
     my $app = OpenQA::FakeApp->new(
         mode     => 'production',
         log_name => 'worker',
         instance => $instance,
         log_dir  => $logdir,
-        level    => $worker_settings->{LOG_LEVEL} // 'debug'
+        level    => $worker_settings->{LOG_LEVEL} // 'info'
```

Comment:
No need to override defaults (as you are already set them here): https://github.com/os-autoinst/openQA/pull/1459/files#diff-774870768a0b33acd9f09ccb5c364ba6R36

Just set the level when `$worker_settings->{LOG_LEVEL}` is set